### DISCUSSION TITLE
Your extended access has ended, although I just bought a subscription

### DISCUSSION THREAD

**Discussion by enedene** [community/community#42573] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Dear all,
I had copilot installed during the trial period and decided to buy a subscription for personal use, 1 year, 100$. I've seen similar issues people had, but that was [marked as solved](https://github.com/community/community/discussions/40022), and none of that works for me.

This is the log message I get:
```[INFO] [auth] [2022-12-24T19:24:00.245Z] Invalid copilot token: missing token: 403 

[ERROR] [default] [2022-12-24T19:24:00.249Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```
When I click on the "Signup for github copilot" warning, browser opens my github acc https://github.com/settings/billing where it is clearly stated that I registered for copilot, 1 year plan, _"You'll be billed $100.00/year after the trial ends on Feb 22, 2023. Cancel anytime."_

I'm not sure if it implicitly associates copilot with my work git account, or something it's unclear how to control this. Global git settings are for my personal and not work account. I'm using Linux Ubuntu and the latest version 1.72.4 of VS code and the latest copilot 1.7.

I have tried restarting, and cleared anything containing `vscode` keyword in `/home/username/.vscode` and `/home/username/.vscode-server` , but the message that my trial period is over continues to appear.

**Comment by jfullstackdev** [comment, marked as answer]
maybe Support can help 

https://support.github.com/contact/account

**Comment by manaberg** [comment]
I have the same problem 👎

**Comment by aescobaricc** [comment]
If you are using visual studio, try log out and login again with your billing account

![Screen Shot 2023-03-12 at 6 10 36 PM](https://user-images.githubusercontent.com/120486257/224573922-8037ebcc-de62-4a2b-baab-7f326e4d1b60.png)


![Screen Shot 2023-03-12 at 6 09 07 PM](https://user-images.githubusercontent.com/120486257/224573846-06a9c01b-eb1e-4be1-ac0d-c6f39eaa1079.png)
